data/courses.json

```
@@ -0,0 +1,112 @@
+{
+  "courses": [
+    {
+      "name": "Deutsch als Fremdsprache",
+      "img": "img/courses/DeutschAbend.png",
+      "header": "\"Wir reden spielend Deutsch - mach mit!\"",
+      "description": "Ihr habt ein wenig Deutsch gelernt und möchtet es nun gerne anwenden und weiter ausbauen? \n In einer lockeren Atmosphäre werden wir viel Sprechen, Diskutieren, Sprach-Spiele anwenden und bestimmt auch mal zusammen deutsche Lieder singen ;-). Hier geht es um viel Praxis! Sprechen kann jeder! Wir werden uns nach Euren Wünschen richten und oft in kleinen Gruppen arbeiten. \n Dieser Kurs richtet sich an Leute, die ihre mündlichen Deutschkenntnisse ausbauen möchten. Grammatikübungen sind nicht ein Bestandteil des Sprach-Cafés.",
+      "additional": "Ergänzend wird gelegentlich ein DAF-Frühstück angeboten, zu dem jeder etwas beisteuern kann, das einfach nur zum Plaudern auf Deutsch bei frischen Schrippen und O-Saft einlädt. Die genauen Termine findet ihr bei Facebook auch bei Meetup",
+      "furtherLink": "https://www.meetup.com/German-Language-Events-Lernwerkstatt-Berlin/",
+      "time": "Immer am letzten Mittwoch des Monats 19-20:30 Uhr"
+    },
+    {
+      "name": "Lernbegleitung",
+      "img": "img/courses/Lernbegleitung.png",
+      "header": "in Mathe, Naturwissenschaften, Englisch, und Deutsch für alle Lernende ab der 7. Klasse.",
+      "description": "Lernbegleitung ist unser schulbegleitendes Angebot. In altersgemischten Kleingruppen von maximal 7 Lernenden werden wöchentlich Inhalte aus der Schule in einer spielerischen und aktiven Lernatmosphäre aufbereitet. Im Fokus steht nicht nur eine Notenverbesserung, sondern vor allem die Begleitung des individuellen Lernprozesses. \n Unsere LernbegleiterInnen kommen aus unterschiedlichen Disziplinen und sind alle Experten in ihren jeweiligen Fächern. Sie bringen zusätzlich zur fachlichen Erfahrung auch viel Welterfahrung mit. Sie sind eingebunden in diverse Aktivitäten innerhalb, sowie außerhalb von Berlin. Diese Erfahrung bereichert die Lernbegleitung.",
+      "additional": "Mit einer kurzen Mail an info@die-lernwerkstatt.org kannst du heute einen Erstermin vereinbaren. Für eine wöchentliche Unterstützung wird eine Mitgliedschaft, mit einem monatlichen Beitrag von 20 Euro, empfohlen.",
+      "furtherLink": "",
+      "time": ""
+    },
+    {
+      "name": "Lerngruppen zu euren Themen",
+      "img": "img/courses/Lerngruppen.png",
+      "header": "",
+      "description": "Für Menschen aller Altersgruppen ist die Lernwerkstatt ein ideeller und realer Raum zum Bearbeiten gemeinsamer Themen. Das kann in Form von Workshops, einmaligen Abendveranstaltungen, fortlaufenden Gruppen, Diskussionen und vielem mehr sein. Ebenso breit gefächert ist das Themenspektrum: Mathematik & Naturwissenschaften, Programmieren & Multimedia, Sprachen, Philosophie, alternative Bildung, u.v.m.",
+      "additional": "",
+      "furtherLink": "",
```

If no link better add just hashtag.
```suggestion
       "furtherLink": "#",
```

---

Done